src/content/docs/configuration/key-binding.mdx

```diff
@@ -18,6 +18,8 @@ filter_mode_shell_up_key_binding = "directory" # or global, host, directory, etc
 
 Our default up-arrow binding can be a bit contentious. Some people love it, some people hate it. Many people who found it a bit jarring at first have since come to love it, so give it a try!
 
+If you want the `enter` key to not execute the command directly like the `tab` key, use the [enter_accept](https://docs.atuin.sh/configuration/config/#enter_accept) config.
```

Comment:
Happy for this to be on this page, but it doesn't make sense right in the middle of the block of text all about the up arrow. It would make sense to give it its own section

Reply:
moved to its own section